lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -43,6 +43,7 @@ class CloudwatchLogsInput < Input
     config_param :end_time, :string, default: nil
     config_param :time_range_format, :string, default: "%Y-%m-%d %H:%M:%S"
     config_param :throttling_retry_seconds, :time, default: nil
+    config_param :max_retry_count, :integer, default: 999 #TODO
```

Comment:
> config_param :max_retry_count, :integer, default: 999 #TODO

Please consider the appropriate default value and remove the TODO comment.

I'm not familiar with CloudWatch.
Can there be a case where a user wants to retry an unlimited number of times, like the current version?

If so, retry should be unlimited when this value is `nil`.
And the default value might be better to be `nil` for compatibility.